pkg/core/chain/chain.go

```diff
@@ -573,7 +572,7 @@ func (c *Chain) requestRoundResults(round uint64) (*block.Block, *block.Certific
 
 	// We wait 5 seconds for a response. We time out otherwise and
 	// attempt catching up later.
-	timer := time.NewTimer(5 * time.Second)
+	timer := time.NewTimer(1 * time.Second)
```

Comment:
parametrizing the timeout might give some control when testing for the resilience of the node on adverse network conditions